src/IQFeed.CSharpApiClient/Streaming/Level1/DynamicFieldset.cs

```diff
@@ -115,9 +115,6 @@ public enum DynamicFieldset
         [FieldsetDescription("Last Time", typeof(TimeSpan))]
         LastTime,
 
-        [FieldsetDescription("Last Trade Date", typeof(DateTime))]
```

Comment:
doesnt exist anymore?

Reply:
Nope. Looks like they changed name to Last Date.

Reply:
on the 6.1 or 6.0 docs?

Reply:
Checked http://www.iqfeed.net/dev/api/docs/docs60/Level1UpdateSummaryMessage.cfm and http://www.iqfeed.net/dev/api/docs/Level1UpdateSummaryMessage.cfm.
Both include Last Date.  Neither include Last Trade Date.

Reply:
sounds good!

Reply:
cool, just add missing tests and we're all set to merge that feature! thank you so much for your patience !

Reply:
Patience? You've been responding in seconds!  Thanks for laying out the design for me!